spyder/plugins/updatemanager/widgets/update.py

```diff
@@ -622,6 +672,51 @@ def __init__(self, icon=None, text=None, parent=None):
         self.setTextFormat(Qt.RichText)
 
 
+class DetailedUpdateMessageBox(UpdateMessageBox, SpyderFontsMixin):
+    def __init__(self, icon=None, text=None, parent=None, details=None):
+        super().__init__(icon=icon, text=text, parent=parent)
+        self.setSizeGripEnabled(True)
+        self.details = None
+        self.setDetailedText(details)
+
+    def setDetailedText(self, details=None):
+        """
+        Overload setDetailedText
+
+        Note: it is critical that QGridLayout.setRowStretch is called after
+        QMessageBox.setDetailedText in order for the stretch behavior to work
+        properly. That is the primary reason for overloading setDetailedText.
```

Comment:
```suggestion
        Override setDetailedText.

        Note: It is critical that QGridLayout.setRowStretch is called after
        QMessageBox.setDetailedText in order for the stretch behavior to work
        properly. That is the primary reason for overriding setDetailedText.
```

I think `override` is the right technical term for redefining a class method (at least that's the one we use in several other places).